WatchApp Extension/Assets.xcassets/Complication.complicationset/Contents.json

```diff
@@ -1,48 +1,53 @@
 {
   "assets" : [
     {
-      "idiom" : "watch",
```

Comment:
Similarly here: this, I think, is only different because of locally building on Xcode 12.5.

Reply:
Is this going to bounce back-and-forth for those not locally building on Xcode 12.5?